lib/engine/g_1856/corporation.rb

```diff
@@ -16,6 +17,7 @@ def initialize(game, sym:, name:, **opts)
         @started = false
         @escrow = nil
         super(sym: sym, name: name, **opts)
+        @all_shares = shares_by_corporation[self].dup
```

Comment:
is this really necesasry? i see it's for total_shares, but can't your 1856 game class count this without having to dup the shares?

Reply:
It's also used in `Engine::Game::G1856::national_issue_shares!` for converting the existing shares to 5% shares and for accessing the shares in `Engine::Game::G1856::national_share_swap` for distributing shares among players. I also have to solve how to give a player a 1-share presidents cert in one of the nationalization edge cases, but that's not solved yet